### PR TITLE
Update Chrome versions for iframe sandbox-allow-modals

### PR DESCRIPTION
**Pull request by felixh10r**
#### Summary
allow-modals support is currently listed as unknown, this PR will add known support for Chrome and related browsers.

#### Test results and supporting details
https://chromestatus.com/feature/4747009953103872
https://googlechrome.github.io/samples/block-modal-dialogs-sandboxed-iframe/index.html

### CODE REVIEW

File: html/elements/iframe.json [hunk 1]
```diff
@@ -912,10 +912,10 @@
...
               "chrome": {
-                "version_added": null
+                "version_added": "46"
               },
               "chrome_android": {
-                "version_added": null
+                "version_added": "46"
               },
               "edge": {
                 "version_added": null
```
Comment (ddbeck) [inline html/elements/iframe.json:921]: This has probably reached some of the other Chrome-derived browsers.

```suggestion
                "version_added": "≤79"
```

And below this, outside of GitHub's suggestion range:

- Opera 33
- Opera for Android 33
- Samsung Internet 5.0

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for this PR, @felixh10r. I see it's your first to the project—welcome! A few suggestions in the line comment. Thanks again!

File: html/elements/iframe.json (hunk 1 above)
Comment (felixh10r) [reply html/elements/iframe.json]: Thanks for the spot-on suggestions, I've added the respective versions.

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**